### ISSUE TITLE
External power source

### ISSUE THREAD

**Issue by eternaleclipse** [wokwi/wokwi-features#338]
It would be nice to have an external power source component with configurable voltage.

Can be useful for playing with different voltages, and also for simulating circuits with different power sources (ex. Arduino controlling 220v power via relay).

**Comment by urish** [comment]
The thing is, Wokwi does not simulate voltages. That being said, I'm seeing users creating a simulation with just electronics (and now also with custom chips too, see #301) and using the ATtiny85 as the "power source" for their circuit. So it might make sense to have some sort of a Power Source component that will be used for these situation.

I'm just not sure what this Power source look be like. Any suggestions?

**Comment by eternaleclipse** [comment]
Yeah, I assume it will mainly be used to simulate circuits with relays (or something that makes sense IRL to use a power source other than what the main board provides).

It can look like a battery, with 2 nodes for + and -, and a user setting for voltage.

**Comment by krishsharma0413** [comment]
bump

**Comment by Mereep** [comment]
bump

**Comment by ChaosNik** [comment]
Bump

**Comment by boboman-1** [comment]
Bump :)

**Comment by alessiadigalo** [comment]
bump

**Comment by HeathersZen** [comment]
I just joined the club today, and instantly found a need for this feature. Designing actual circuits beyond the 3.3 the ESP32 provides means there must be a power supply somewhere...

**Comment by djedu28** [comment]
@HeathersZen 
It turns out that the [Wokwi still does not simulate voltages](https://github.com/wokwi/wokwi-features/issues/338#issuecomment-1104994565), you can even (optionally) not connect VCC and GND and the circuit in the WOKWI will work normally.

in wokwi we have the symbols `VCC` and `GND`:
![image](https://github.com/user-attachments/assets/e3b38ca0-5054-4b96-b838-142e9d1e106b)

you can use them in the circuit, instead of an external source
![image](https://github.com/user-attachments/assets/c2ef77cd-661d-48f8-8a1d-8482c48dfebc)

**Comment by cgten-uk** [comment]
Sorry to bump this, but the VCC symbol is pointless in this context if you can't configure the voltage, or even see what it is.

From a user perspective, this seems rather simple:  Add a VCC symbol as above, then click on it to set the required voltage.

In many of my projects, I run my MCU on different voltages to the components connected to it.  This is less likely when using 5v Arduinos, but for ESP32s and other MCUs that run on 3.3v logic, it becomes a problem.

Even if the ability to simulate voltage behind the scenes doesn't exist in Wokwi at this stage, that VCC symbol isn't even documented anywhere, it's just... there.
